code/LeftAndMain.php

```diff
@@ -281,16 +244,35 @@ class LeftAndMain extends Controller implements PermissionProvider
     private static $frame_options = 'SAMEORIGIN';
 
     /**
-     * The configuration passed to the supporting JS for each CMS section includes a 'name' key
-     * that by default matches the FQCN of the current class. This setting allows you to change
-     * the key if necessary (for example, if you are overloading CMSMain or another core class
-     * and want to keep the core JS - which depends on the core class names - functioning, you
-     * would need to set this to the FQCN of the class you are overloading).
+     * The urls used for the links in the Help dropdown in the backend
+     *
+     * Set this to `false` via yml config if you do not want to show any help links
      *
      * @config
-     * @var string|null
+     * @var array
      */
-    private static $section_name = null;
+    private static $help_links = [
+        'CMS User help' => 'https://userhelp.silverstripe.org/en/5',
+        'Developer docs' => 'https://docs.silverstripe.org/en/5/',
```

Comment:
```suggestion
        'CMS User help' => 'https://userhelp.silverstripe.org/en',
        'Developer docs' => 'https://docs.silverstripe.org/en/',
```

Reply:
Or /6/

Reply:
We'll need to update a bunch of references like this, we should probably just do them all together. But I'll make this change to avoid ping pong.
Going with an explicit `6` so people don't get docs for `7` in their `6` sites when we get to that point.